packages/calypso-products/src/plans-list.tsx

```diff
@@ -1445,6 +1445,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_STYLE_CUSTOMIZATION,
 		FEATURE_LIVE_CHAT_SUPPORT,
 		FEATURE_WORDADS,
+		FEATURE_STATS_PAID,
```

Comment:
This is /plans/<site> page.

<img width="1775" alt="Screenshot 2024-01-05 at 3 08 37 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/f13cdcc2-a580-4e20-a64b-e8db68ac9502">